core/arch/arm/kernel/thread_optee_smc.c

```diff
@@ -249,7 +249,8 @@ bool thread_disable_prealloc_rpc_cache(uint64_t *cookie)
 		if (threads[n].rpc_arg) {
 			*cookie = mobj_get_cookie(threads[n].rpc_mobj);
 			mobj_put(threads[n].rpc_mobj);
-			threads[n].rpc_arg = NULL;
+			threads[n].rpc_arg = 0;
```

Comment:
typo: `= NULL;` as in original implementation.

Reply:
You're right. I'll update `__thread_std_smc_entry()` above.